Assets/Mirage/Runtime/PlayerSpawner.cs

```diff
@@ -26,6 +26,11 @@ public class PlayerSpawner : MonoBehaviour
         [FormerlySerializedAs("playerPrefab")]
         public NetworkIdentity PlayerPrefab;
 
+        /// <summary>
+        /// Opertionally delay the spawn.
```

Comment:
It won't delay it, it will prevent it completely no? Maybe a mention/comment somewhere to call that new public method when this is turned off. Otherwise loogs good.